column/plugins/callback/progress.py

```diff
@@ -0,0 +1,32 @@
+import logging
+
+import column
+
+LOG = logging.getLogger(__name__)
+
+
+class AnsibleTrackProgress(column.AnsibleCallback):
+    def __init__(self, *args, **kwargs):
+        super(self.__class__, self).__init__(*args, **kwargs)
+        self.total_plays = 0
+        self.finished_plays = 0
+        self.playbook = None
+        self.progress = 0
+
+    def run_on_playbook_start(self, playbook):
+        self.playbook = playbook._file_name
```

Comment:
This reminds me that we need to refactor the callbacks so that ansible attributes don't need to be accessed like this example.